src/commands/build.ts

```diff
@@ -32,6 +32,21 @@ export class BuildCommand implements Command {
 
   args = [
     {
+      name: 'js-minify',
```

Comment:
I really dislike the `.` in the command line flag, a dash feels much more natural and doesn't break the expected CLI dash-case naming convention. We can still map easily regardless of `.` or `-`.